publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/config/SecurityConfig.java

```diff
@@ -74,20 +108,54 @@ public Integer getTimeOut() {
     @Autowired
     private CustomAuthenticationEntryPoint customAuthenticationEntryPoint;
 
-    @Autowired
-    protected void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+    @Bean
+    public UserCache userCache() {
+        if (cacheManager().getCache("authenticationCache") == null) {
```

Comment:
Maybe I don't understand it, but 
  cacheManager().getCache("authenticationCache") == null
should alway be true. But maybe I don't understand something...

I was unable to run it in debugger. We should have a call and discuss that.